x-pack/plugins/security_solution/public/common/components/exceptions/add_exception_modal/index.tsx

```diff
@@ -65,7 +66,7 @@ interface AddExceptionModalProps {
     nonEcsData: TimelineNonEcsData[];
   };
   onCancel: () => void;
-  onConfirm: () => void;
+  onConfirm: (didCloseAlert?: boolean) => void;
```

Comment:
It seems from all the uses of `onConfirm` that the boolean argument is always present, do we need the `?`?